src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -632,6 +634,10 @@ static void sigsegv_handler(int code, siginfo_t *siginfo, void *context)
         {
             if (GetCurrentPalThread())
             {
+#if defined(TARGET_TVOS)
+                (void)!write(STDERR_FILENO, StackOverflowMessage, sizeof(StackOverflowMessage) - 1);
+                PROCAbort(SIGSEGV, siginfo);
```

Comment:
This likely needs to be resolved in a different way.

Reply:
It seems like tvOS just cannot handle stack overflow gracefully. We probably would not even get this far.

Reply:
I don't think tvOS could hit this path. I assume it uses the exception port mechanism instead of signals like macOS, or am I mistaken?

Reply:
Ah, that's surprising, ok.